### PR TITLE
Updated minimum deployment to Swift 5.9/macOS 13

### PR DESCRIPTION
**Pull request by dimitribouniol**
Updated minimum deployment to Swift 5.9/macOS 13 to support things like Duration and other modern niceties.

### CODE REVIEW

**Review by marius-se — APPROVED** [review]
LGTM 🚀

**Comment by 0xTim** [comment]
@swift-server-bot please test

**Comment by 0xTim** [comment]
@dimitribouniol could you update https://github.com/swift-server/webauthn-swift/tree/main/docker as well to point to 5.9 from 5.7?

**Comment by dimitribouniol** [comment]
Fixed!

**Comment by dimitribouniol** [comment]
@swift-server-bot please test

**Comment by dimitribouniol** [comment]
(worth a try 😛)

File: docker/docker-compose.2204.57.yaml
Comment (0xTim) [inline docker/docker-compose.2204.57.yaml:1]: Small nit - could you rename the file to reflect that its now 5.9?

**Review by 0xTim — CHANGES_REQUESTED** [review]

File: docker/docker-compose.2204.57.yaml
Comment (dimitribouniol) [reply docker/docker-compose.2204.57.yaml:1]: Done! Does the `2204` part need to change as well? Not _entirely_ sure what it refers to, but happy to bring it forward to `2403` if it's a date or something.

Comment (0xTim) [reply docker/docker-compose.2204.57.yaml:1]: It's the ubuntu version, which we will still use so all good on that front

**Review by 0xTim — APPROVED** [review]
Thanks!

**Comment by 0xTim** [comment]
@swift-server-bot test

**Comment by 0xTim** [comment]
@swift-server-bot please test

**Comment by yim-lee** [comment]
@swift-server-bot add to allowlist

**Comment by yim-lee** [comment]
@swift-server-bot test this please

**Comment by yim-lee** [comment]
Updated CI. "pull request validation (5.9)" replaces "pull request validation (5.7)".

**Comment by dimitribouniol** [comment]
Thanks @yim-lee!

**Pull request merged**